### DISCUSSION TITLE
How to Customize Colors for General Site

### DISCUSSION THREAD

**Discussion by JRHacka** [chipzoller/hugo-clarity#340] [Q&A]
Started playing around with the override.sass file and the variable.sass file but can't seem to get the colors to stick. Just want to change some of the general site colors, not specific items. I see the color hex code and think I understand what's happening but am clearly missing something. Any advice would be extremely welcome.

**Comment by chipzoller** [comment]
@rootwork and @onweru

**Comment by onweru** [comment]
@JRHacka,

There are two ways to go about it
1. Use the [overrides sass](https://github.com/chipzoller/hugo-clarity/blob/master/assets/sass/_override.sass) file. [Paste](https://github.com/chipzoller/hugo-clarity/blob/829f6816d4b82f5aa02b5740ec7899a746a99dfe/assets/sass/_variables.sass#L25-L74) into it and edit your colors.
2. duplicate the [variables sass](https://github.com/chipzoller/hugo-clarity/blob/master/assets/sass/_variables.sass) file. and edit it accordingly. 

Note: you cannot and should not edit the theme directly. The path and the name of the sass file should mirror the name and path on the theme.
